pages/ja/lb4/README_todo-tutorial.md

```diff
@@ -0,0 +1,95 @@
+# @loopback/example-todo
```

Comment:
This one is tricky because the English version is coming from the README file of [examples/todo](https://github.com/strongloop/loopback-next/blob/master/examples/todo/README.md). But _I think_ we can put the translated version in this location. 

@nabdelgadir, could you please confirm? In this case, we need to add the front matter for this page. And also rename it to `todo-tutorial.md` to match the original file name.

Reply:
I think there's already a todo tutorial in Japanese [here](https://loopback.io/doc/ja/lb4/todo-tutorial.html) where the README was copied.